.coveragerc

```diff
@@ -72,6 +72,7 @@ omit =
     homeassistant/components/bh1750/sensor.py
     homeassistant/components/bitcoin/sensor.py
     homeassistant/components/bizkaibus/sensor.py
+    homeassistant/components/blastbot_cloud/*
```

Comment:
This integration provides a config flow.

The config flow itself should not be excluded from the coverage. The configuration flow requires tests with 100% coverage.